float64[] task_pose_reference